src/ImpactX.cpp

```diff
@@ -297,9 +301,9 @@ namespace impactx
                   px.push_back(ipx);
                   py.push_back(ipy);
                   pt.push_back(ipt);
-                  amrex::PrintToFile("initial_beam.txt") << ix << " " << iy << " ";
-                  amrex::PrintToFile("initial_beam.txt") << it << " " << ipx << " ";
-                  amrex::PrintToFile("initial_beam.txt") << ipy << " " << ipt << "\n";
+                  amrex::PrintToFile("diags/initial_beam.txt") << ix << " " << iy << " ";
```

Comment:
It would be more efficient to merge the three calls to PrintToFile into one.

Reply:
Yes, that makes sense.  Line 304 of course needs to be continued to include it, ipx, ipy, and ipt.

Reply:
Thanks, good catch! :)